test/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.t.sol

```diff
@@ -1268,7 +1268,7 @@ contract FM_BC_Bancor_Redeeming_VirtualSupplyV1Test is ModuleTest {
     function testInternalSetVirtualIssuanceSupply_WorksGivenIssuanceSupplyBiggerThanZero(
         uint _newSupply
     ) public {
-        vm.assume(_newSupply != 0);
+        vm.assume(_newSupply > 10);
```

Comment:
Same as above, should we use `INITIAL_ISSUANCE_SUPPLY` here just to be clear of where this number comes from?

Reply:
Not doing this, not needed.